data-access/hdp_data_access.ipynb

```diff
@@ -0,0 +1,236 @@
+   "id": "6ccad77a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat = intake.open_esm_datastore(\"https://cadcat.s3.amazonaws.com/histwxstns/era-hdp-collection.json\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "989e64b5",
+   "metadata": {},
+   "source": [
+    "Next, view the catalog in table format. You can inspect the first few rows by calling `.head()` on the table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0389d0d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Access catalog as dataframe and inspect the first few rows\n",
+    "cat_df = cat.df\n",
+    "cat_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e57cfbd5",
+   "metadata": {},
+   "source": [
+    "View all the weather station networks by using the following code "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c743e382",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# See all network options \n",
+    "cat_df[\"network_id\"].unique()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4864dd8c",
+   "metadata": {},
+   "source": [
+    "You can also filter the catalog to see all stations within a network"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "687f661b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_network = \"ASOSAWOS\"\n",
+    "cat_df[cat_df[\"network_id\"] == my_network]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ddfe314",
+   "metadata": {},
+   "source": [
+    "You can subset the catalog and read in the Zarrs as `xarray.Dataset` objects using the method shown below. To change the data downloaded, simply modify the inputs in the dictionary `query`. These inputs must correspond to valid options in the catalog. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a169749a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set your query here \n",
+    "query = {\n",
+    "    \"network_id\": \"ASOSAWOS\",  # Name of the network\n",
+    "    \"station_id\": [\"ASOSAWOS_A0002694297\",\"ASOSAWOS_A0704900320\",\"ASOSAWOS_72020200118\"] # List of stations to get data for \n",
+    "}\n",
+    "\n",
+    "# Subset catalog \n",
+    "cat_subset = cat.search(**query)\n",
+    "\n",
+    "# View the data you've selected before downloading\n",
+    "cat_subset.df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31bc9a9b",
+   "metadata": {},
+   "source": [
+    "Then, you can download all the files. The files will be downloaded as a dictionary with the following format: \n",
+    "```\n",
+    "{ <string ID of data> : <xarray.Dataset for that ID> }\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a6eaf70",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get dataset dictionary \n",
+    "dsets = cat_subset.to_dataset_dict(\n",
```

Comment:
This failed for me ("Access Denied") but it seems like that might be expected behavior if the data's not public yet?

Reply:
Oooo no that's not desired, it should be public. I'll look into it, thanks!

Reply:
Issue resolved